tests/test_hvac_csc.py

```diff
@@ -136,6 +87,17 @@ async def test_version(self):
     async def test_bin_script(self):
         await self.check_bin_script(name="HVAC", index=None, exe_name="run_hvac.py")
 
+    async def _verify_evt_deviceEnabled(self, subsystem):
+        # Default mask indicating that the devices, that always are enabled,
+        # are enabled.
+        device_mask = 0b111
+        device_id = DeviceId[subsystem]
+        deviceId_index = list(DeviceId).index(device_id)
```

Comment:
Please look for this everywhere and replace with a dict.

Reply:
Done